api/src/pcapi/scripts/generate_missing_profile_completion_fraud_checks.py

```diff
@@ -0,0 +1,64 @@
+from pcapi.core.fraud import models as fraud_models
+from pcapi.core.users.models import EligibilityType
+from pcapi.core.users.models import User
+from pcapi.core.users.models import UserRole
+from pcapi.models import db
+
+
+BATCH_SIZE = 1000
+
+
+def generate_missing_profile_completion_fraud_checks(dry_run: bool = True) -> None:
+    print("Fetching users...")
+    users_with_profile_completed_but_no_profile_fraud_check = User.query.filter(
+        User.city.isnot(None),
+        User.activity.isnot(None),
+        User.firstName.isnot(None),
+        User.lastName.isnot(None),
+        ~User.beneficiaryFraudChecks.any(
+            fraud_models.BeneficiaryFraudCheck.type == fraud_models.FraudCheckType.PROFILE_COMPLETION
+        ),
+    ).with_entities(User.id)
+    user_ids = [user_id_tuple[0] for user_id_tuple in users_with_profile_completed_but_no_profile_fraud_check.all()]
+    fraud_checks_created_count = 0
+    start_index = 0
+    print("Creating fraud checks...")
+    while start_index < len(user_ids):
```

Comment:
tu peux mettre 1 print de plus pour voir où t'en es
print("{start_index} users treated out of {len(user_ids)})